### PR TITLE
feat: Adding tags to all resources that support them

### PR DESCRIPTION
**Pull request by harsha-chamarthi**
Tags seem to be missing for all created resources using terraform. I have added tags to all the resources that support them.
Please check and approve

### CODE REVIEW

File: modules/web-app/variables.tf [hunk 1]
```diff
@@ -78,3 +78,9 @@ variable "log_level" {
 variable "name_suffix" {
   type = string
 }
+
+variable "tags" {
+  description = "Map of tags that will be added to created resources."
+  type        = map(string)
+  default     = {}
+}
```
Comment (mitchell-liatrio) [inline modules/web-app/variables.tf:86]: Add newline at end of file

File: variables.tf [hunk 2]
```diff
@@ -167,3 +167,9 @@ variable "log_level" {
   type    = string
   default = "Information"
 }
+
+variable "tags" {
+  description = "Map of tags that will be added to created resources."
+  type        = map(string)
+  default     = {}
+}
```
[the remark this answers is not in the document]
Comment (harsha-chamarthi) [reply variables.tf]: Added newline to both the files and fixed indentation

**Comment by harsha-chamarthi** [comment]
> @harsha-chamarthi love the contribution. Couple of minor edits/asks before merging these in. Can you run a `terraform fmt --recursive` to fix indentation issues here and update the following files to have newlines at the end of file
> 
> * modules/web-app/variables.tf
> * variables.tf

@mitchell-liatrio I have a question, can we go ahead and add the additional tags to the runner VM's being created on top of the existing tag runnerIdentifierLabel? 
ref: https://github.com/liatrio/terraform-azure-github-runner/blob/main/app/runner-controller/azure/index.js#L149

**Comment by mitchell-liatrio** [comment]
> > @harsha-chamarthi love the contribution. Couple of minor edits/asks before merging these in. Can you run a `terraform fmt --recursive` to fix indentation issues here and update the following files to have newlines at the end of file
> 
> > 
> 
> > * modules/web-app/variables.tf
> 
> > * variables.tf
> 
> 
> 

@harsha-chamarthi absolutely.  You can either do it as part of this PR or I'll create and issue and get it added early next week
> @mitchell-liatrio I have a question, can we go ahead and add the additional tags to the runner VM's being created on top of the existing tag runnerIdentifierLabel? 
> 
> ref: https://github.com/liatrio/terraform-azure-github-runner/blob/main/app/runner-controller/azure/index.js#L149

@har

**Review by mitchell-liatrio — APPROVED** [review]
LGTM 🚀

**Comment by harsha-chamarthi** [comment]
> > > @harsha-chamarthi love the contribution. Couple of minor edits/asks before merging these in. Can you run a `terraform fmt --recursive` to fix indentation issues here and update the following files to have newlines at the end of file
> > 
> > 
> > > 
> > 
> > 
> > > * modules/web-app/variables.tf
> > 
> > 
> > > * variables.tf
> 
> > @mitchell-liatrio I have a question, can we go ahead and add the additional tags to the runner VM's being created on top of the existing tag runnerIdentifierLabel?
> > ref: https://github.com/liatrio/terraform-azure-github-runner/blob/main/app/runner-controller/azure/index.js#L149
> 
> @harsha-chamarthi absolutely. You can either do it as part of this PR or I'll create and issue and get it added early next week

@mitchell-liatrio Thanks i will create an issue for getting lables added to the runner

**Pull request merged**